examples/models/llava/README.md

```diff
@@ -1,20 +1,88 @@
 ## Summary
-In this example, we initiate the process of running multi modality through ExecuTorch.
-- Demonstrate how to export the image encoder model in the [LLava](https://github.com/haotian-liu/LLaVA) multimodal model.
-- Provide TODO steps on how to use the exported .pte file and the existing [exported Llama2 model](https://github.com/pytorch/executorch/tree/main/examples/models/llama2), to build the multimodal pipeline.
+LLaVA is the first multi-modal LLM ExecuTorch supports. In this directory, we
+- Host a model definition for LLaVA.
+- Demonstrate how to export [LLavA](https://github.com/haotian-liu/LLaVA) multimodal model to a .pte file.
+- Provide a C++ runner that loads the .pte file, the tokenizer and an image, then generate responses based on user prompt.
```

Comment:
Where is the image from? Can user generate image.pt from there own images?

Reply:
Will add a util in next PR